### ISSUE TITLE
Merge SlicerFreeSurfer and SlicerFreeSurferCommands and move into the Slicer GitHub organization?

### ISSUE THREAD

**Issue by benzwick** [PerkLab/SlicerFreeSurfer#15]
Should we merge [SlicerFreeSurfer](https://github.com/PerkLab/SlicerFreeSurfer) and [SlicerFreeSurferCommands](https://github.com/SlicerCBM/SlicerFreeSurferCommands), and move them to the Slicer GitHub organization?

Please see the discussion [here](https://github.com/Slicer/ExtensionsIndex/pull/1943) for more details.

One thing to consider is that the `SlicerFreeSurfer` extension doesn't have any external dependencies as far as I'm aware, whereas the `SlicerFreeSurferCommands` extension depends on FreeSurfer which needs to be installed and configured by the user, takes up about 14 GB of disk space, and can only be installed on Linux/Unix OS (on Windows WSL can be used).

**Comment by jcfr** [comment]
Instead moving the project `SlicerFreeSurferCommands` to the PerkLab GitHub organization, per suggestion of @pieper , the idea would be the following:

* bundle the modules inside the `PerkLab/SlicerFreeSurfer` extension
* make it clear that these commands are only available if freesurfer is installed.

> That is, make it clear to the user at runtime, rather than in documentation, what features would be available if freesurfer is installed along with pointers to the installation instructions (even a button that opens the web page).

**Comment by Sunderlandkyl** [comment]
I agree, adding the SlicerFreeSurferCommands to SlicerFreeSurfer sounds like the best approach.

**Comment by lassoan** [comment]
@benzwick would you mind creating a pull request that adds module from https://github.com/SlicerCBM/SlicerFreeSurferCommands to this extension?
